improver/cli/estimate_emos_coefficients_from_table.py

```diff
@@ -153,7 +153,13 @@ def process(
         EstimateCoefficientsForEnsembleCalibration,
     )
 
-    filters = [("diagnostic", "==", diagnostic)]
+    # Load forecasts from parquet file filtering by diagnostic and blend_time.
+    cycletimes = pd.date_range(
+        end=pd.Timestamp(cycletime) - pd.Timedelta(1, unit="days"),
+        periods=int(training_length),
+        freq="D",
+    ).tz_localize(None)
```

Comment:
Did you find this didn't work with tzawareness?

Reply:
Removing `tz_localize(None)` gives the error: `TypeError: Cannot compare tz-naive and tz-aware timestamps`.

Reply:
Yes, I think I had similar, might want to comment as it is tzaware data (when extracted!)

Reply:
I've added a comment.